---
title: "Screw: tools and hardware for building reproducible single-cell epigenomics workflows"
author: "Kieran O'Neill, Chelsey Fang, Benjamin Decato, Alexander Goncearenco, Azhar Khandekar, Ben Busby, Aly Karsan"
output: "beamer_presentation"
toc: "false"
---

## Reproducible Research 

Reproducible research requires having the exact **data**, **code** and **software** used to perform an analysis.

![](figures/dockstore_logos.png)

Image from dockstore.org, used under Apache-2.0

## Single-cell Whole Genome Bisulfite Seq 

![](figures/nrg3980-f1.jpg)

Schwartzman et al (2015) Nature Reviews in Genetics (used by permission) 


## Single-cell Whole Genome Bisulfite Seq Data

![](figures/hoxa9_meth.png)


## Preprocessing: Example problem -- lack of standards!

![](figures/data_types.pdf)



## Output: Distance matrix and clustering


![](figures/farlik_heatmap.pdf)


## Status and Plans


- So far we just have one example data set, with some basic preprocessing and heatmap generation. 

- But about half a dozen SC-WGBS (or SC-RRBS) public data sets now exist.
- We intend to use Screw to perform a meta-analysis of these.
- More vis, more tools, differential methylation calling.


## Acknowledgements

**NCBI Genomics Hackathon**: Benjamin Decato, Alexander Goncearenco, Azhar Khandekhar, Ben Busby (hackathon organiser)

**Summer intern**: Chelsey Fang

![](figures/people.pdf)

`https://github.com/Epigenomics-Screw`
Poster: A-153 (Sunday 6pm)
